src/main/java/seedu/address/logic/parser/AttendanceCommandParser.java

```diff
@@ -1,37 +1,59 @@
 package seedu.address.logic.parser;
 
-import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ATTENDANCE;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_ATTENDANCE_COMMAND_FORMAT;
 
-import seedu.address.commons.core.index.Index;
-import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.logic.commands.AttendanceCommand;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.logic.commands.AttendanceAddCommand;
+import seedu.address.logic.commands.AttendanceDeleteCommand;
+import seedu.address.logic.commands.AttendanceMarkCommand;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.HelpCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-//import seedu.address.model.attendance.Attendance;
+
 
 
 /**
- * Parses input arguments and creates a new {@code AttendanceCommand} object
+ * Parses input arguments and creates new AttendanceParsers object
```

Comment:
AttendanceParsers object? Not sure what you mean here. AttendanceCommand object seems fine